content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst

```diff
@@ -4,34 +4,118 @@ Inventory adjustments
+   move with `0.00` :guilabel:`Quantity Done` is recorded in the product's inventory adjustment
+   history, so it should be set to reflect the actual quantity counted.
 
-To add an inventory adjustment line, click :guilabel:`Create` and fill in the :guilabel:`Product`
-and :guilabel:`Counted Quantity` fields. Then click :guilabel:`Save`.
+To the right of the :guilabel:`Counted Quality` column, the :guilabel:`Scheduled Date` and
+:guilabel:`User` can also be changed via their respective drop-down menus. Changing the scheduled
+date changes the date that the inventory adjustment should be processed on, and selecting a
+responsible user assigns a user to the specific inventory adjustment (for traceability purposes).
 
-At this stage, the count is recorded but not yet *applied*. Meaning the quantity on hand is not yet
-updated to match the new :guilabel:`Counted Quantity`.
+Once all changes have been made to the new inventory adjustment line, click away from the line.
+Doing so saves the adjustment, and moves the line to the top of the page.
 
-To apply the new :guilabel:`Counted Quantity`, click :guilabel:`Apply` on the line or the
-:guilabel:`Apply All` button on the top of the page. If :guilabel:`Apply All` is clicked, a
-reference or reason can be entered before clicking :guilabel:`Apply`.
+If the :guilabel:`Counted Quantity` is greater than the :guilabel:`On Hand Quantity`, the value in
+the :guilabel:`Difference` column is **green**. If the :guilabel:`Counted Quantity` is lesser than
+the :guilabel:`On Hand Quantity`, the value in the :guilabel:`Difference` column is **red**. If the
+quantities match, and haven't been changed at all, no value appears in the :guilabel:`Difference`
+column.
 
-.. image:: count_products/apply-inventory-adjustment.png
+.. image:: count_products/count-products-difference-column.png
+   :align: center
+   :alt: Difference column on inventory adjustments page.
+
+At this stage, the count (:dfn:`inventory adjustment`) is recorded, but not yet applied. This means
+that the quantity on hand before the adjustment has not yet been updated to match the new, real
+counted quantity.
+
+There are two ways to apply the new inventory adjustment. The first way is to click the
+:guilabel:`Apply` button on the line at the far right of the page. The second way is to click the
+checkbox on the far left of the line. Doing so reveals new button options at the top of the page,
+one of which is an :guilabel:`Apply` button. Clicking this button instead causes an
+:guilabel:`Inventory Adjustment Reference / Reason` pop-up to appear.
+
+From this pop-up menu, a reference or reason can be assigned to the inventory adjustment. By
+default, this is pre-populated with the date the adjustment is being made on, but can be changed to
+reflect whatever reference or reason is desired.
+
+Once ready, click :guilabel:`Apply` to apply the inventory adjustment.
+
+.. image:: count_products/count-products-apply-inventory-adjustment.png
    :scale: 60%
```

Comment:
No need to have this :scale: tag on a properly sized image, that being said, this image is too wide at 1215px wide. Please resize to be under 933px ideally 768 🙂
```suggestion
```